app/Http/Controllers/SocialImageController.php

```diff
@@ -2,13 +2,13 @@
 
 namespace App\Http\Controllers;
 
-use App\Actions\GenerateSocialShareImage;
+use App\Jobs\GenerateSocialShareImage;
 use App\Models\Article;
 
 class SocialImageController extends Controller
 {
     public function __invoke(GenerateSocialShareImage $generateImage, Article $article)
     {
-        return $generateImage($article);
+        return $this->dispatchSync(new $generateImage($article));
```

Comment:
This can just be `new GenerateSocialShareImage($article)` without the need to inject the job through the invoke method.